ui/app/src/Utils/Types.elm

```diff
@@ -30,10 +29,6 @@ type alias Label =
     ( String, String )
 
 
-type alias ApiData a =
-    ApiResponse Http.Error a
```

Comment:
Since that thing was a library, it had to be more generic. We don't need more than String for error at the moment. Changing this to smth else later won't be a problem.